analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -14,6 +14,7 @@
       <SonarAnalyzerRegressionTestRunning Condition="$(SonarAnalyzerRegressionTestRunning) == ''">false</SonarAnalyzerRegressionTestRunning>
       <SonarProjectOutputPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)</SonarProjectOutputPath>
       <SonarProjectConfigFilePath>$(SonarProjectOutputPath)\SonarProjectConfig.xml</SonarProjectConfigFilePath>
+      <SonarFilesToAnalyzePath>$(SonarProjectOutputPath)\FilesToAnalyze.txt</SonarFilesToAnalyzePath>
```

Comment:
could you add a comment why the `SonarProjectConfig` and `FilesToAnalyze` are needed?  Something like

"We need to create some files which are normally created by the Scanner for .NET"